test/DynamoCoreTests/Models/RunSettingsTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Dynamo.Models;
+using NUnit.Framework;
+
+namespace Dynamo.Tests.ModelsTest
+{
+    [TestFixture]
+    class RunSettingsTest
+    {
+        /// <summary>
+        /// This test method will execute the RunSettings constructor without parameters
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void RunSettingsConstructorTest()
+        {
+            //Act
+            //Ejecuting the constructor without parameters
```

Comment:
`Executing`

Reply:
@QilongTang 
I fixed spelling in the commit: https://github.com/DynamoDS/Dynamo/pull/10657/commits/e4cb6087f2d7f55cffab39b432839f630fe10bea